src/PlaneWaveBasis.jl

```diff
@@ -12,8 +12,10 @@ More generally, a kpoint is a block of the Hamiltonian;
 eg collinear spin is treated by doubling the number of kpoints.
 """
 struct Kpoint{T <: Real}
+    model::Model{T}
     spin::Symbol                     # :up, :down, :both or :spinless
     coordinate::Vec3{T}              # Fractional coordinate of k-Point
+    coordinate_cart::Vec3{T}         # Cartesian coordinate of k-Point
```

Comment:
cart =>cartesian ?

Reply:
I actually think coordinate is too verbose :p

Reply:
But cartesian is shorter, so that's ok, no ;)